tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -0,0 +1,140 @@
+#
+# Check for:
+#
+# ironicAPIReadyCount: 1
+# ironicConductorReadyCount: 1
+# apiEndpoints:
+#     ironic:
+#       admin: http://ironic-admin-openstack.apps-crc.testing
+#       internal: http://ironic-internal-openstack.apps-crc.testing
+#       public: http://ironic-public-openstack.apps-crc.testing
+# we should see and validate reason and status as below on all the DBReady, DBSyncReady, BootstrapReady, DeploymentReady, ExposeServiceReady, InputReady, IronicAPIReady, IronicConductorReady, IronicRabbitMqTransportURLReady, ServiceConfigReady etc.
+# reason: Ready
+#  status: "True"
+
+apiVersion: ironic.openstack.org/v1beta1
+kind: Ironic
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"ironic.openstack.org/v1beta1","kind":"Ironic","metadata":{"annotations":{},"name":"ironic","namespace":"openstack"},"spec":{"containerImage":"quay.io/openstack-k8s-operators/ironic-operator-index:latest","customServiceConfig":"[DEFAULT]\ndebug = true\n","databaseInstance":"openstack","databaseUser":"ironic","ironicAPI":{"containerImage":"quay.io/tripleozedcentos9/openstack-ironic-api:current-tripleo","replicas":1},"ironicConductor":{"containerImage":"quay.io/tripleozedcentos9/openstack-ironic-conductor:current-tripleo","pxeContainerImage":"quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo","replicas":1,"storageClass":"local-storage","storageRequest":"10G"},"ironicInspector":{"containerImage":"quay.io/tripleozedcentos9/openstack-ironic-inspector:current-tripleo","pxeContainerImage":"quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo","replicas":1},"secret":"osp-secret","serviceUser":"ironic","storageClass":"local-storage"}}
+  creationTimestamp: "2023-02-14T14:50:07Z"
+  finalizers:
+  - Ironic
+  generation: 1
+  name: ironic
+  namespace: openstack
+  resourceVersion: "75736"
+  uid: 17c6557d-8e10-46fc-8ce7-9ba67bab2b1b
+spec:
+  customServiceConfig: |
+    [DEFAULT]
+    debug = true
+  databaseInstance: openstack
+  databaseUser: ironic
+  ironicAPI:
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-api:current-tripleo
+    customServiceConfig: '# add your customization here'
+    databaseUser: ironic
+    passwordSelectors:
+      database: IronicDatabasePassword
+      service: IronicPassword
+    replicas: 1
+    rpcTransport: json-rpc
+    serviceUser: ironic
+    standalone: false
+  ironicConductor:
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-conductor:current-tripleo
+    customServiceConfig: '# add your customization here'
+    databaseUser: ironic
+    passwordSelectors:
+      database: IronicDatabasePassword
+      service: IronicPassword
+    pxeContainerImage: quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo
+    replicas: 1
+    rpcTransport: json-rpc
+    serviceUser: ironic
+    standalone: false
+    storageClass: local-storage
```

Comment:
This could change in the future, so it might be a good idea to avoid checking for it. See: https://github.com/openstack-k8s-operators/mariadb-operator/pull/70

Reply:
done